src/app/clients/ActionBar.cy.tsx

```diff
@@ -9,7 +9,7 @@ describe("Clients - Action Bar", () => {
         {
             primaryKey: "primaryKey1",
             addressPostcode: "AB1 2CD",
-            collectionCentre: "Centre 1",
+            collectionCentreAcronym: "Centre 1",
```

Comment:
is acronym not supposed to be an abbreviation? this ("centre 1") looks more like name to me